src/util.ts

```diff
@@ -0,0 +1,280 @@
+import * as stencila from '@stencila/schema'
+import Ajv from 'ajv'
+import betterAjvErrors from 'better-ajv-errors'
+import fs from 'fs-extra'
+import globby from 'globby'
+import produce from 'immer'
+import path from 'path'
+import { parse as parsePerson } from './person'
+
+const built = path.join(
+  path.dirname(require.resolve('@stencila/schema')),
+  'built'
+)
+
+// Load all schemas for use in by Ajv validator
+const schemas = globby
+  .sync(path.join(built, '*.schema.json'))
+  .map(file => fs.readJSONSync(file))
```

Comment:
Yes you're right. This could have contributed to that sluggishness. In a previous version we had lazy loading of schema via Ajv's https://github.com/epoberezkin/ajv#asynchronous-validation. This was always intended to be a temporary use of `sync` to avoid some of the complications introduced by `async`. Will create a new issue.